### PR TITLE
fix typo and simplified boolean logic

### PR DESCRIPTION
**Pull request by Thaina**
Also add JSDoc intellisense for webxr, work in vscode when trying to check the type and get suggestion for property

### CODE REVIEW

**Comment by De-Panther** [comment]
Thanks for the PR.
Can you please separate the commits to different PRs?
I'm not sure about the JSDoc one, but I do want to approve the typo and logic changes

**Comment by Thaina** [comment]
Alright

File: Build/Build/Build.framework.js [hunk 1]
```diff
@@ -996,8 +996,8 @@ setTimeout(function () {
         if (xrData.controllerA.updatedProfiles == 1 || xrData.controllerB.updatedProfiles == 1)
         {
           var inputProfiles = {};
-          inputProfiles.conrtoller1 = xrData.controllerA.profiles;
-          inputProfiles.conrtoller2 = xrData.controllerB.profiles;
+          inputProfiles.controller1 = xrData.controllerA.profiles;
+          inputProfiles.controller2 = xrData.controllerB.profiles;
```
Comment (De-Panther) [inline Build/Build/Build.framework.js]:
```suggestion
          inputProfiles.conrtoller1 = xrData.controllerA.profiles;
          inputProfiles.conrtoller2 = xrData.controllerB.profiles;
```

**Review by De-Panther — APPROVED** [review]
Thanks!

**Comment by Thaina** [comment]
Should the `Build` folder be added to gitignore too?

Or just change it into `Builds` folder maybe? And retroactively remove it from the history

**Comment by De-Panther** [comment]
If you want to build something for test, use 'Builds' or revert changes.
The 'Build' folder is for the demo.
I update it for releases

**Pull request merged**